modules/ossm-rn-new-features.adoc

```diff
@@ -19,11 +19,101 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
+== New features {SMProductName} 2.2
+
+This release of {SMProductName} adds new features and enhancements, and is supported on OpenShift Container Platform 4.9, and 4.10.
+
+=== Component versions included in {SMProductName} version {SMProductVersion}
+
+|===
+|Component |Version
+
+|Istio
+|1.12.7
+
+|Envoy Proxy
+|1.20.4
+
+|Jaeger
+|1.30.2
+
+|Kiali
+|1.48.0.16
+|===
+
+=== `WasmPlugin` API
+This release changes the implementation of `WebAssembly` extensions to use the `WasmPlugin` API instead of the `ServiceMeshExtention` API, which has been deprecated.
+
+=== ROSA support
+This release introduces service mesh support for Red Hat OpenShift on AWS (ROSA), including multi-cluster federation.
+
+=== `istio-node` DaemonSet renamed
+This release, the `istio-node` DaemonSet is renamed to `istio-cni-node` to match the name in upstream Istio.
+
+=== Envoy sidecar networking changes
+Istio 1.10 updated Envoy to send traffic to the application container using `eth0` rather than `lo` by default.
+
+=== Service Mesh Control Plane 1.1
+This release marks the end of support for service mesh control planes based on Service Mesh 1.1 for all platforms.
+
+=== Istio 1.12 Support
+
+{SMProductShortName} 2.2 is based on Istio 1.12, which brings in new features and product enhancements. While many Istio 1.12 features are supported, the following exceptions should be noted:
```

Comment:
`While many Istio 1.12 features are supported, the following exceptions should be noted:`
Is not as clear as I'd prefer.  Maybe revise as
"While many Istio 1.12 features are supported, the following unsupported features should be noted:"
(I really want to get the word "unsupported" in there for clarity)

Reply:
Change made